doc/compiler/aot/SymbolValidationManager.md

```diff
@@ -58,18 +58,18 @@ guaranteed to be compatible in the current environment.
 
 However, since `J9Class`es are artifacts of the current environment, how
 does the SVM enable a new environment to validate it? The core idea is to
-associate unique IDs to each new symbol the compiler gets a hold of. Thus,
-when a symbol is acquired by asking a query using a different symbol, the
-validation record that gets generated refers to these two symbols by their
-IDs, which are environment agnostic. In the load run, the AOT load
-infrastructure uses these IDs to build up its table of symbols that are
+associate unique IDs to each new value the compiler gets a hold of. Thus,
+when a value is acquired by asking a query using a different value, the
+validation record that gets generated refers to these two values by their
+IDs (symbols), which are environment agnostic. In the load run, the AOT load
+infrastructure uses these IDs to build up its table of value that are
```

Comment:
This should be "it's table of _values_ [plural] that are"

Reply:
Updated in force push https://github.com/eclipse-openj9/openj9/compare/bf8608af47dbd1df69cf073d45cf70d166e29a25..f23a15e3597cc6e11fdaf9712f42e58e6c9ebcff